### ISSUE TITLE
Few Missing Entities

### ISSUE THREAD

**Issue by nomanurrehman** [loverajoel/spotify-js#26]
@loverajoel 

I think since there has been no activity in this package, it is missing a few entities like episodes and shows.

Also, I could not find a handler for search where you could perform a generic search for all entities.

I would be happy to add these in and submit a pull request.

**Comment by loverajoel** [comment]
Hi @nomanurrehman, sorry for the delay! Yes, a few entities are missing. Last year I started to work on a rewrite for spotify-sdk. The idea was to simplify the API. Unfortunately, this year is impossible for me to spend more time on it, and I had to put it on hold.
You can take a look here https://github.com/loverajoel/spotify-sdk/tree/v2.0.

Do you think we should add the new entities there? If you are up to help me to finish the migration of the new version I can add you as an admin. 🙏

**Comment by nomanurrehman** [comment]
@loverajoel 

Yes! That would be really nice. I would love to continue moving this forward and taking it to the finish line.

Out of curiousity, since you are busy, I would guess if you had to look at my pull request, it would take some time? I am okay with that, just to be sure that we will be working together on this going forward? Or perhaps more contributors would help?

If you are looking to me to take charge then that's okay with me too :)
